plugin/al/lib/AL_USDMaya/AL/usdmaya/nodes/ProxyShape.h

```diff
@@ -749,6 +762,9 @@ class ProxyShape
         return m_unloadedProxyShapes;
     }
 
+    AL_USDMAYA_PUBLIC
+    static void loadUnloadedProxyShapes();
```

Comment:
The definition is missing.

Reply:
sorry about that, it looks like it was just an accidental add part of another old feature that is no longer needed. Removed!